CRM/Upgrade/Incremental/php/FiveSixtySeven.php

```diff
@@ -56,7 +56,29 @@ public function upgrade_5_67_alpha1($rev): void {
     $this->addTask('Add cache_fill_took column to Group table', 'addColumn', 'civicrm_group', 'cache_fill_took',
       'DOUBLE DEFAULT NULL COMMENT "Seconds taken to fill smart group cache, not always related to cache_date"',
       FALSE);
-    $this->addTask('Update civicrm_mailing_event_queue to permit deleting records from civicrm_mailing_job', 'updateMailingEventQueueTable', 'updateMailingEventQueueTable');
+    $this->addTask('Update civicrm_mailing_event_queue to permit deleting records from civicrm_mailing_job', 'updateMailingEventQueueTable');
```

Comment:
This is unrelated - I wasn't sure about this, but I think the second argument here is not required?